source/Calamari/Kubernetes/KubernetesContextScriptWrapper.cs

```diff
@@ -52,11 +53,13 @@ public bool IsEnabled(ScriptSyntax syntax)
             {
                 environmentVars = new Dictionary<string, string>();
             }
-            
+
+            var kubectl = new Kubectl(variables, log, commandLineRunner, workingDirectory, environmentVars);
```

Comment:
For wrappers I'm newing up kubectl here instead of inside the components (eg: inside `SetupKubectlAuthentication`). The affective behaviour is the same for the deployment scripts where kubectl instances are created numerous times, each time printing messages to the task log.